rviz_python_tutorial/myviz.py

```diff
@@ -18,6 +18,10 @@
 ## python_qt_binding internally, so you should use it here as well.
 from python_qt_binding.QtGui import *
 from python_qt_binding.QtCore import *
+try:
+    from python_qt_binding.QtWidgets import *
+except ImportError:
+    pass
```

Comment:
What's the rationale behind doing a silent pass on import failure here? In what situation does this fail?

Reply:
QApplication is in QtWidgets in Qt5. My intent is to import QtWidgets only when using Qt5